src/plugins/data_views/server/fetcher/index_patterns_fetcher.ts

```diff
@@ -137,23 +137,21 @@ export class IndexPatternsFetcher {
   async validatePatternListActive(patternList: string[]) {
     const result = await Promise.all(
       patternList
-        .map((pattern) =>
-          this.elasticsearchClient.count({
-            index: pattern,
-          })
-        )
-        .map((p) =>
-          p.catch((e) => {
-            if (e.body.error.type === 'index_not_found_exception') {
-              return { body: { count: 0 } };
-            }
-            throw e;
-          })
-        )
+        .map(async (index) => {
+          const searchResponse = await this.elasticsearchClient.fieldCaps({
```

Comment:
changed from `count` to `fieldCaps` because with `count` there appears to be a race condition where count returns 0. changing to `fieldCaps` consistently returns the correct indices